### ISSUE TITLE
New-UDInputAction -RedirectUrl -inifinte loop

### ISSUE THREAD

**Issue by Phyziik** [ironmansoftware/universal-dashboard#98]
Whenever I use UDInputAction to redirect to a Dynamic page, I just get a infinite spinning widget.

My test code : 
```
$homePage = New-UDPage -Name "Home" -Content {
   New-UDRow -Columns {
      New-UDColumn -DebugEndpoint -Endpoint {
         New-UDInput -Title "Monitored computer" -SubmitText "Monitor" -Content {
            New-UDInputField -Type "textbox" -Name "ComputerName" -Placeholder "Computer Name"
         } -Endpoint {
            param($ComputerName)
            
            New-UDInputAction -RedirectUrl "/ComputerMonitoring/$ComputerName"
         }
      }
   }
}
$redirector = New-UDPage -Url "/ComputerMonitoring/:testString" -Endpoint {
   param($testString)
   New-UDCard -Title "Test Card" -Content {"$testString"}
}
$MyDashboard = New-UDDashboard -Title "Hello, World" -Pages @($homePage,$redirector) -EndpointInitializationScript {
   Enable-UDLogging -Level Error -FilePath ".\debug_logger.txt" -Console
}
Start-UDDashboard -Port 1000 -Dashboard $MyDashboard -AutoReload
```

**Comment by apetitjean** [comment]
I think this is the same issue I opened here #96

**Comment by adamdriscoll** [comment]
@Phyziik Arnaud was correct. This was the same issue and has been resolved in 1.6.0-beta3.
